Dockerfile-cct

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.9 as go-builder
```

Comment:
Could you move this to docker folder like https://github.com/uber/cadence/pull/793?